src/client/components/Checkbox/Checkbox.js

```diff
@@ -10,18 +10,21 @@ export type Props = {
   onChange?: (e: SyntheticInputEvent) => any,
   name: string,
   disabled?: boolean,
+  style?: any,
 }
 
 class Checkbox extends Component {
   constructor(props: Props) {
     super(props)
     this.state = {
       checked: this.props.checked,
+      disabled: this.props.disabled ? this.props.disabled : false,
```

Comment:
But the `disabled` was in ChangesetList's state before not in the Checkbox. It should work the following way: 
1 - `checkbox` triggers change event and send its state to the check handler with `checked/unchecked` value and changeset `ID` value
2 - `checkbox lists` handles checkbox change event and maintain checked values in its state 

CheckboxList -> `handleChange`
3 - if two checkboxes selected set `state.disabled` to `True` which basically means we should rerender all checkboxes and make just two of them(which selected)  `enabled`, others - `disabled` (this will force rerendering of checkbox list) 
4 - if one checkbox unselected and checkbox list `state.disabled` is `True`, set `state.disabled` to `False` - this should force all checkboxes to rerender and will enable them all, and update the changesets IDs in `CheckboxList` state
5 - if one checkbox is selected - save the changeset ID to `CheckboxList` state

Preferably rerender `CheckboxList` only when `state.disabled` is changed.